[2024-02-26T14:39:10.739+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-01-02T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509
[2024-02-26T14:39:10.740+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-01-02T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-01-02T00:30:00+00:00/task_id=print_current_date_with_jinja'
[2024-02-26T14:39:10.766+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-01-02T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509
[2024-02-26T14:39:10.766+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-01-02T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-01-02T00:30:00+00:00/task_id=print_current_date_with_jinja'
[2024-02-26T14:39:10.969+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: python_dag_with_jinja.print_current_date_with_jinja scheduled__2024-01-02T00:30:00+00:00 [queued]>
[2024-02-26T14:39:10.978+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: python_dag_with_jinja.print_current_date_with_jinja scheduled__2024-01-02T00:30:00+00:00 [queued]>
[2024-02-26T14:39:10.979+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-02-26T14:39:10.999+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): print_current_date_with_jinja> on 2024-01-02 00:30:00+00:00
[2024-02-26T14:39:11.005+0000] {standard_task_runner.py:57} INFO - Started process 1443 to run task
[2024-02-26T14:39:11.008+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'python_dag_with_jinja', 'print_current_date_with_jinja', 'scheduled__2024-01-02T00:30:00+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/04-python-operator-with-jinja.py', '--cfg-path', '/tmp/tmpez6afjz7']
[2024-02-26T14:39:11.010+0000] {standard_task_runner.py:85} INFO - Job 105: Subtask print_current_date_with_jinja
[2024-02-26T14:39:11.053+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-01-02T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509
[2024-02-26T14:39:11.053+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-01-02T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-01-02T00:30:00+00:00/task_id=print_current_date_with_jinja'
[2024-02-26T14:39:11.056+0000] {task_command.py:410} INFO - Running <TaskInstance: python_dag_with_jinja.print_current_date_with_jinja scheduled__2024-01-02T00:30:00+00:00 [running]> on host 2fe49a870f57
[2024-02-26T14:39:11.136+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='heumsi' AIRFLOW_CTX_DAG_ID='python_dag_with_jinja' AIRFLOW_CTX_TASK_ID='print_current_date_with_jinja' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:30:00+00:00'
[2024-02-26T14:39:11.138+0000] {logging_mixin.py:150} INFO - 2024-01-02는 화요일입니다
[2024-02-26T14:39:11.139+0000] {python.py:183} INFO - Done. Returned value was: 2024-01-02
[2024-02-26T14:39:11.163+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=python_dag_with_jinja, task_id=print_current_date_with_jinja, execution_date=20240102T003000, start_date=20240226T143910, end_date=20240226T143911
[2024-02-26T14:39:11.180+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-02-26T14:39:11.196+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
